lib/resources/Base.js

```diff
@@ -102,16 +122,19 @@ export default class {
    * instance.urlFor();                            // `/admin/products.json`
    * instance.urlFor(1);                           // `/admin/products/1.json`
    * instance.urlFor("1");                         // `/admin/products/1.json`
+   * instance.urlFor(1, { type: "test" })          // `/admin/products/1.json?type=test
```

Comment:
Yeah...sad that I wrote this code, and I didn't realize there is already a test for this :disappointed: